doc/autoscale/experiment/README.md

```diff
@@ -143,33 +143,61 @@ serving together. This case is a very typical scenario for large enterprises and
 
 ## Reproduce the experiment
 
-- Configure kubectl on your host
 - Prepare
-    1. Configure kubectl 
-    1. Configure paddlectl
-    1. Submit the TrainingJob controller with YAML file
+    1. Configure kubectl and paddlectl on your host.
+    1. Submit the TrainingJob controller with the YAML file.
     ```bash
     > git clone https://github.com/PaddlePaddle/cloud.git && cd cloud
     > kubectl create -f k8s/controller/trainingjob_resource.yaml
     > kubectl create -f k8s/controller/controller.yaml
     ```
-- Test Case1
-    1. Run the TestCase1 for serval passes with bash scripts`./control_case.1.sh`:
+- Run the Test Case
+    1. Run the TestCase1 or TestCase2 for serval passes with the bash script `./run.sh`:
+        For example, run TestCase1 for 10 passes and 10 jobs:
         ```bash
-        > cd cloud/doc/autoscale/experiment/python
-        > ./control_case1.sh --help
-        > usage: control_case1.sh <action>
-            action[required]: str[start|stop], will start or stop all the jobs.
-          env var:
-            JOB_COUNT[optional]:             int, The number of submiting jobs, defualt is 1.
-            FAULT_TOLERANT[optional]:   str[ON|OFF], whether a fault-tolerant job,default is OFF.
-            PASSES[optional]:           int, The number of run passes.
-            DETAILS[optional:           str[ON|OFF], print detail monitor information.
+        > cd cloud/doc/autoscale/experiment
+        > AUTO_SCALING=OFF PASSES=5 JOB_COUNT=40 ./run.sh start case1
         ```
-        For example, run TestCase1 for 10 passes and 10 jobs:
+        Or submit an auto-scaline training job
+        > cd cloud/doc/autoscale/experiment
         ```bash
-            > PASSES=10 JOB_COUNT=10 ./control_case1.sh start
+        > AUTO_SCALING=ON PASSES=5 JOB_COUNT=40 ./run.sh start case1
         ```
+        Or run the TestCase2 with 5 jobs:
+        ```bash
+        > JOB_COUNT=5 ./run.sh start case2
+        ```
+	1. Get the time series data.
+		The time serise data will be appended in the file `./out/mnist-case[1|2]-pass[0-9].log`
+		as the following format:
+
+
+		```
+		0,2.11,0,3,0,0,0,0,0|0|0,0.00|0.00|0.00
```

Comment:
I saw duplicated timestamps from different line of records from @helinwang 's change, but here is no more. was this a bug?